src/grammar/grammar.ohm

```diff
@@ -119,6 +120,8 @@ Tact {
     StatementWhile = while "(" Expression ")" "{" Statement* "}"
     StatementRepeat = repeat "(" Expression ")" "{" Statement* "}"
     StatementUntil = do "{" Statement* "}" until "(" Expression ")" ";"
+    StatementTry = try "{" Statement* "}" ~catch --simple
+                 | try "{" Statement* "}" catch "(" id ")" "{" Statement* "}" --withCatch
```

Comment:
why don't we make the catch part optional (`?`) this should help deduplicate the grammar a bit

Reply:
I thought about that at first, but to match the style and logic of grammar definitions of other statements, I did it as I did.

Reply:
Agree with @Gusarich here. If that's needed, we may later enhance the whole grammar file at once